src/main/kotlin/app/utils/PasswordHelper.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.utils
+
+import org.apache.commons.codec.digest.DigestUtils
+import java.io.IOError
+
+/**
+ * Class for hashing and reading passwords.
+ */
+object PasswordHelper {
+    fun hashPassword(password: String): String {
+        return if (password.isEmpty()) {
```

Comment:
We don't want an empty password to have hash, so it is the realization of a hash function.